### ISSUE TITLE
Unable to change or set any passwords

### ISSUE THREAD

**Issue by bstone108** [bpatrik/pigallery2#831]
**Note:** Please always try reproducing the issue on the `:edge` build. There is a chance that it was already fixed. If it's an UI issue, you can try reporducing it on the [demo page](https://pigallery2.onrender.com/). 

----------------------------------------------------------------------------------

**Describe the bug**

unable to click username to open menu, unable to find a password change option in settings, and there's no enforced users in config.  the options simply do not exist.  I'm stuck with the default admin/admin because there's no way to change it provided anywhere in the interface.

**Photo/video (optional) that causes the bug**

If error is in connection with a particular photo/video, It would help a lot with the debugging if you can attach that file too.

**Environment (please complete the following information):**
 - OS: docker, linux, unraid
 - Browser any, tried them all

**Used app version**:

- docker-2.0.0

**Comment by bpatrik** [comment]
Yeah that is not implemented.

Feel free to do so and send a PR.

For now you can create an other account and delete the default admin

-- Sorry for being brief, sent from my phone.

**Comment by bstone108** [comment]
Yeah I figured it out right after opening ticket.  I just haven't had
chance to update it yet.  Thanks

**Comment by bpatrik** [comment]
This should be added now to the settings.
